src/untils/db.js

```diff
@@ -0,0 +1,10 @@
+import 'dotenv/config';
+import { Sequelize } from 'sequelize';
+
+export const client = new Sequelize({
+  host: process.env.DB_HOST,
+  username: process.env.DB_USER,
+  password: process.env.DB_PASSWORD,
+  database: process.env.DB_DATABASE,
+  dialect: 'postgres',
+});
```

Comment:
Consider adding additional configuration options such as `logging: false` to disable SQL query logging in production, and `pool` settings to manage database connections more efficiently.